### PR TITLE
Fix bug 1183337 - changes wording and add link to howtos in the banner

### PR DESCRIPTION
**Pull request by dirtycoder**

### CODE REVIEW

File: kuma/wiki/jinja2/wiki/document.html [hunk 1]
```diff
@@ -262,14 +262,24 @@
...
-                  <div class="warning warning-review"><p>{% trans %}
-                    This article is in need of a technical review.
-                  {% endtrans %}</p></div>
+                  <div class="warning warning-review">
+                    <p>
+                      {% trans %}This article needs a technical review.{% endtrans %}
+                      <a href="https://developer.mozilla.org/docs/MDN/Contribute/Howto/Do_a_technical_review">
+                        {% trans %}How you can help.{% endtrans %}
+                      </a>
+                    </p>
```
Comment (willkg) [inline kuma/wiki/jinja2/wiki/document.html]: Jinja2 trans tag allows for variables, so you could do this like so:

```
{% trans url="https://developer.mozilla.org/docs/MDN/Contribute/HowTo/Do_a_technical_review" %}
  This article needs a technical review. <a href="{{ url }}">How you can help.</a>
{% endtrans %}
```

Ditto for the other text changes.

**Comment by willkg** [comment]
This was marked as a work in progress, but it looks like it hasn't gotten any activity in a month. What's left to do here?

**Comment by dirtycoder** [comment]
Hey @willkg, I was left with little free time, sorry about that.
I'll take a final look at it this weekend.

**Comment by dirtycoder** [comment]
@stephaniehobson I'm just notifying you about this PR because I'll need to talk to you later.

**Comment by stephaniehobson** [comment]
Sorry I didn't get to this this week. I'll review it Monday.

File: kuma/wiki/jinja2/wiki/document.html [hunk 2]
```diff
@@ -262,14 +262,22 @@
...
+                  </div>
                 {% endif %}
                 {% if document.current_revision.needs_editorial_review %}
-                  <div class="warning warning-review"><p>{% trans %}
-                    This article is in need of an editorial review.
-                  {% endtrans %}</p></div>
+                  <div class="warning warning-review">
+                    <p>
+                      {% trans url="https://developer.mozilla.org/docs/MDN/Contribute/Howto/Do_an_editorial_review" %}
+                        This artible needs an editorial review. <a href="{{ url }}">How you can help.</a>
```
Comment (stephaniehobson) [inline kuma/wiki/jinja2/wiki/document.html]: Typo for "article".

Comment (dirtycoder) [inline kuma/wiki/jinja2/wiki/document.html]: Not a problem at all.
Thanks for pointing the typo.

File: kuma/wiki/jinja2/wiki/document.html [hunk 3]
```diff
@@ -262,14 +262,22 @@
 
               {% if document.current_revision %}
                 {% if document.current_revision.needs_technical_review %}
-                  <div class="warning warning-review"><p>{% trans %}
-                    This article is in need of a technical review.
-                  {% endtrans %}</p></div>
+                  <div class="warning warning-review">
+                    <p>
+                      {% trans url="https://developer.mozilla.org/docs/MDN/Contribute/HowTo/Do_a_technical_review" %}
```
Comment (stephaniehobson) [inline kuma/wiki/jinja2/wiki/document.html]: Please make this a relative URL (remove the "https://developer.mozilla.org").

File: kuma/wiki/jinja2/wiki/document.html [hunk 4]
```diff
@@ -262,14 +262,22 @@
...
+                    </p>
+                  </div>
                 {% endif %}
                 {% if document.current_revision.needs_editorial_review %}
-                  <div class="warning warning-review"><p>{% trans %}
-                    This article is in need of an editorial review.
-                  {% endtrans %}</p></div>
+                  <div class="warning warning-review">
+                    <p>
+                      {% trans url="https://developer.mozilla.org/docs/MDN/Contribute/Howto/Do_an_editorial_review" %}
```
Comment (stephaniehobson) [inline kuma/wiki/jinja2/wiki/document.html]: Please make this a relative URL (remove the "https://developer.mozilla.org").

**Comment by stephaniehobson** [comment]
I'm not sure how to test the tagging is working right. Can one of the back-end folk weigh in on the revision.py changes? @jezdez @robhudson or @jwhitlock?

File: kuma/wiki/jinja2/wiki/document.html (hunk 3 above)
Comment (dirtycoder) [inline kuma/wiki/jinja2/wiki/document.html]: Oh, I can't believe I made this. :astonished: 
I'll fix this later.

**Comment by stephaniehobson** [comment]
I had a more in-depth look at how this is working and I'm no longer concerned about the scope of the changes.

Sorry for the long review @dirtycoder, many of our back-end resources got moved to another team :(

**Comment by dirtycoder** [comment]
No problem @stephaniehobson, thanks! :)

**Pull request merged**